### PR TITLE
Feature/issue 424

### PR DESCRIPTION
**Pull request by ear7h**
Simply skips assignment if the variable is named "_"

### CODE REVIEW

**Comment by st0012** [comment]
@ear7h Can you add some tests on VM side as well?

File: compiler/bytecode/expression_generation.go [hunk 1]
```diff
@@ -140,6 +140,11 @@ func (g *Generator) compileAssignExpression(is *InstructionSet, exp *ast.AssignE
 	}
 
 	for i, v := range exp.Variables {
+		if v.TokenLiteral() == "_" {
+			//skip assignment
+			goto L
```
Comment (st0012) [inline compiler/bytecode/expression_generation.go]: I prefer using other flow control mechanism other than `goto`

**Comment by st0012** [comment]
@ear7h I hope you can also add integration tests in VM's `TestAssignmentEvaluation`

**Comment by ear7h** [comment]
@st0012 doh

**Comment by st0012** [comment]
@ear7h This is quite useful, thanks!

**Pull request merged**